### ISSUE TITLE
Bug with spaces in filenames

### ISSUE THREAD

**Issue by nathanielvirgo** [purescript/spago#822]
I just did the following:

* installed purescript and spago via npm
* created a folder `/Users/nathaniel/Code/purescript learning/test-project-1` and cd'd into it
* ran `spago init`
* ran `spago build`
* ran `spago test`

I was surprised to get the following error:

```
[info] Build succeeded.
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '/Users/nathaniel/Code/purescript'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
[error] Tests failed: exit code: 1
```

but I guessed correctly that the problem was the space in the name of the enclosing folder. Changing its name from `purescript learning` to `purescript-learning` gives the expected result, i.e. it reports success.

This is only a minor inconvenience, but it seems a bit silly to fail due to spaces in a filename.

**Comment by f-f** [comment]
Thanks for reporting this! It's definitely a bug, and it comes from the way that we are calling `node` (i.e. I suspect that something is not quoted [when we put together the script to call](https://github.com/purescript/spago/blob/7728ce42a1c66968cce7a72dbb6ab05ea0904d9f/src/Spago/Build.hs#L335)) in `spago run`/`spago test`

**Comment by jbrains** [comment]
A note to the folks who need it. If you're bitten by this, but don't want to rename your project directory, you can run these commands:

```
# Equivalent to `spago run`
node -e "require('./output/Main/index').main()"

# Equivalent to `spago test`, as long as you use the default test module name.
# If you're a spago/purescript beginner, then you are probably using the default test module name. :)
node -e "require('./output/Test.Main/index').main()"
```

**Comment by f-f** [comment]
Fixed in #846
